simul/platform/cliutils.go

```diff
@@ -33,16 +36,24 @@ func Scp(username, host, file, dest string) error {
 // Rsync copies files or directories to the remote host. If the DebugVisible
 // is > 1, the rsync-operation is displayed on screen.
 func Rsync(username, host, file, dest string) error {
-	addr := host + ":" + dest
+	h, p, err := net.SplitHostPort(host)
+	if err != nil && strings.Contains(err.Error(), "missing port in address") {
+		p = "22"
+		err = nil
+	}
+	if err != nil {
+		return err
+	}
```

Comment:
The succession of 2 ifs, assigning `err = nil` instead of using an else, looks somewhat weird. 

```suggestion
	if err != nil && strings.Contains(err.Error(), "missing port in address") {
		p = "22"
	} else if err != nil {
		return err
	}
```

err will be reassigned by whoever uses it later.

Reply:
Why isn't this
```go
if err != nil {
	if !strings.Contains(err.Error(), "missing port in address") {
		return err
	}
	p = "22"
}
```